lua/telescope/previewers.lua

```diff
@@ -373,7 +373,11 @@ previewers.help = defaulter(function(_)
 
         local escaped = vim.fn.escape(entry.value, special_chars)
         local tagfile = vim.fn.expand("$VIMRUNTIME") .. '/doc/tags'
+        local old_tags = vim.o.tags
+        vim.o.tags = tagfile
         local taglist = vim.fn.taglist('^' .. escaped .. '$', tagfile)
+        vim.o.tags = old_tags
```

Comment:
I didn't think we need to set old_tags if we pass the tag file to `taglist`. When I was testing this locally, I didn't seem to need to do this.

Reply:
Perhaps you have something else dynamically seeing your tags var?